packages/blocks/src/multiple-choice/edit.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useClientId } from '@crowdsignal/hooks';
+
+/**
+ * Style dependencies
+ */
+import { MultipleChoiceWrapper } from './styles/multiple-choice';
+
+const ALLOWED_ANSWER_BLOCKS = [
+	'core/image',
+	'core/paragraph',
+	'crowdsignal-forms/poll-answer',
+];
+
+const MultipleChoice = ( props ) => {
+	const { attributes, setAttributes } = props;
+
+	useClientId( props );
+
+	const handleChangeQuestion = ( question ) => setAttributes( { question } );
+
+	const handleChangeNote = ( note ) => setAttributes( { note } );
+
+	return (
+		<MultipleChoiceWrapper>
```

Comment:
Note: in terms of styling, we might want to just create some kind of general `QuestionWrapper` component that's just this wrapper with all the styles.  

I imagine it will/should probably be the same for all question blocks. Added to c/BQqTn3fQ-tr.

Reply:
Ah, didn't think of it this way, but it makes more sense to just wrap the answers. On the other hand, using InnerBlocks prevents us from forcibly assigning a class to those answers. If you look at the style, you'll see I'm using what seems like an autogenerated class name for the answers: https://github.com/Automattic/crowdsignal-ui/pull/44/files/3b16011ff77431c3db0230fb31bb186926f3c712#diff-d9e43fcae2fb4e379078aa8aafb0a3a7d40b0f244b5b965c5da6defb201a9898R8